fedora_messaging/cli.py

```diff
@@ -452,3 +458,33 @@ def record(exchange, queue_name, routing_key, app_name, limit, file):
     _consume(
         exchange, queue_name, routing_key, messages_recorder.collect_message, app_name
     )
+
+
+URL_TEMPLATE = "https://apps.fedoraproject.org/datagrepper/id?id={}&is_raw=true"
+
+
+@cli.command()
+@click.argument("message_id")
+@click.option("--datagrepper_url", help=_datagrepper_help, default=URL_TEMPLATE)
```

Comment:
Oh yeah one minor thing: the convention is that option words are separated by dashes, not underscores. So it should be `--datagrepper-url`. Click will convert that to underscores for the variable name.